src/devices/Mcp3428/Mcp3426.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Device.I2c;
+
+namespace Iot.Device.Mcp3428
+{
+    public class Mcp3426 : Mcp3428
+    {
+        public const int Address = 0x68;
+
+        /// <inheritdoc />
+        public Mcp3426(I2cDevice i2CDevice) : base(i2CDevice, 2)
+        {
+        }
+
+        /// <inheritdoc />
+        public Mcp3426(I2cDevice i2CDevice, ModeEnum mode = ModeEnum.Continuous, ResolutionEnum resolution = ResolutionEnum.Bit12, GainEnum pgaGain = GainEnum.X1) : this(i2CDevice)
+        {
+            SetConfig(0, mode: mode, resolution: resolution, pgaGain: pgaGain);
+        }
+
+#pragma warning disable RCS1163 // Unused parameter.
```

Comment:
I think you should either create a base class for these two and put static method in only in the derived classes or remove this method - in the current state this is really confusing

Reply:
From the above selection I'm not sure what you refer to. Most likely it's about the I2CAddressFromPins in Mcp3426, this was always a weak point of the design. I didn't want to implement Mcp3426/7 until I realized that they only differ in channel count and address configuration. May introduce base class for all three.